app/services/waste_carriers_engine/notify/base_send_email_service.rb

```diff
@@ -3,6 +3,9 @@
 module WasteCarriersEngine
   module Notify
     class BaseSendEmailService < BaseService
+      include WasteCarriersEngine::ApplicationHelper
```

Comment:
@irisfaraway Mixed into the base class. Do you think that's the best place or would it sit better in the class that uses it?

Reply:
If it gets reused by multiple child classes (which I think it does?) then I'm fine with having it in here.